components/android/app/build.gradle

```diff
@@ -61,6 +63,7 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.android.support:multidex:1.0.3'
```

Comment:
What is the reasons of adding this library? please specify using `because` method

Reply:
I was facing an issue while using cloud_firestore: 
`Execution failed for task ':app:transformDexArchiveWithExternalLibsDexMergerForDebug'.`
So I followed this solution: https://stackoverflow.com/a/54680454/11545939

Reply:
ok it make sense here